.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "pytorch_xla",
+  "image": "us-central1-docker.pkg.dev/tpu-pytorch/docker/development:tpu",
+  "runArgs": [
+    "--privileged"
+  ],
+  "forwardPorts": [
+    4200
+  ],
+  "initializeCommand": "docker pull us-central1-docker.pkg.dev/tpu-pytorch/docker/development:tpu",
+  "mounts": [
+    "source=${localEnv:HOME}/.ssh,target=/home/vscode/.ssh,type=bind"
```

Comment:
This shouldn't be necessary. VSCode will forward your local SSH agent's keys automatically: https://code.visualstudio.com/remote/advancedcontainers/sharing-git-credentials#_using-ssh-keys

Reply:
Removed, thanks!